actions/playerHeroesActions.js

```diff
@@ -38,6 +38,6 @@ export const getPlayerHeroes = (playerId, numMatches, host = HOST_URL) => (dispa
   dispatch(getPlayerHeroesRequest());
   return fetch(`${host}${url(playerId, numMatches)}`, { credentials: 'include' })
     .then(response => response.json())
-    .then(json => dispatch(getPlayerHeroesOk(json.matches)))
+    .then(json => dispatch(getPlayerHeroesOk(json.slice(0, 10))))
     .catch(error => dispatch(getPlayerHeroesError(error)));
```

Comment:
We shouldn't be slicing this here, if they want more hero data, then we have to make a separate request to retrieve the data we just threw away here. It's better to just limit the number of heroes we return to that specific table. I'm not sure if that functionality is built in yet though. Just something to keep in mind